### ISSUE TITLE
Broken link in README.md

### ISSUE THREAD

**Issue by kristiank** [dita-for-small-teams/dita-for-small-teams#2]
The link provided is broken: http://d4st.org/ prefixing with www ends up in an GitHub 404 page.

**Comment by drmacro** [comment]
Yes, I'm still working out the details of how to host d4st.org--my initial attempt at redirect to the github pages didn't work.

**Comment by 4everJang** [comment]
Hi Eliot,

I have a D4st site ready to host on your domain, based on a ditamap with all content in dita topics and doing a live transform. When I mentioned I would put the content on GitHub, I meant the dita topics and map, not the php that transforms them into responsive html5.

I have been very busy with other work in the past month, so I have not tried pulling individual files from a GitHub repo yet, but that should be doable, I guess. The d4st domain does not need to redirect to GitHub as a site, I don't think. Let me know what your plans are. We can do a Skype session this week if you want. Not today, as I was wiped off my feet by the flu. I should be better tomorrow or Thursday.

Ciao

Jang



Jang F.M. Graat
Smart Information Design
Amsterdam, Netherlands
Cell: +31 646 854 996

 <http://amsterdamdatascience.nl/>
 <https://www.facebook.com/4everJang>
 <https://twitter.com/4everJang> <https://nl.linkedin.com/in/4everjang> <https://github.com/4everJang>
 <https://github.com/4everJang>

**Comment by drmacro** [comment]
OK. I can set up a new hosting service through Network Solutions unless you prefer a different provider.

**Comment by 4everJang** [comment]
Didn’t you already have a hosting service for the domain ? I thought you were using One.com <http://one.com/> but maybe you only grabbed the domain name without hosting.

**Comment by drmacro** [comment]
No, I've been using Network Solutions as my hosting service (e.g., for contrext.com)

**Comment by 4everJang** [comment]
Let me first run a couple of checks to see whether you really need a separate hosting service for the site. I have a couple domains that I can use to do some testing. As the content is supposed to be on GitHub anyway, there might not be any need for a real hosted solution, just an alias might do.
 
]
